pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -1088,7 +1088,7 @@ public Subscription getSubscription() {
 
     private int addAndGetUnAckedMsgs(Consumer consumer, int ackedMessages) {
         int unackedMsgs = 0;
-        if (Subscription.isIndividualAckMode(subType)) {
+        if (isDurable && Subscription.isIndividualAckMode(subType)) {
```

Comment:
If we create a no-durable consumer under a persistent topic, the field `isDurable` is also `false`, right? How could we cover this case?

Reply:
So we need to check if the topic is persistent.